src/js/operations/Geo.js

```diff
@@ -0,0 +1,33 @@
+L.DNC.Geo = L.Class.extend({
+    includes: L.Mixin.Events,
+
+    options: {},
+
+    buffer: {
+        maxFeatures: 1,
+        additionalArgs: 0.1,
+        description: 'Calculates a buffer for input features for a given radius. Units supported are miles, kilometers, and degrees.',
+        parameters: [
+            {
+                name: 'distance',
+                description: 'Distance to draw the buffer.',
+                type: 'number',
+                default: 10
+            },
+            {
+                name: 'unit',
+                type: 'select',
+                description: '',
+                options: ['miles', 'feet', 'kilometers', 'meters', 'degrees'],
+                default: 'miles'
+            }
+        ]
+    },
+
+    union: {
+        minFeatures: 2,
+        maxFeatures: 2,
+        description: 'Takes two polygons and returns a combined polygon. If the input polygons are not contiguous, this function returns a MultiPolygon feature.'
+    }
```

Comment:
Thanks @alukach! I'm going to make sure I've got tests down and ready before merging this - but hopefully ready before this coming Wednesday (#57) what do you think?